test/test_dynamic_shapes.py

```diff
@@ -12,12 +12,13 @@ def test_simple_expand(self):
     size2 = 2
     t1 = torch.zeros([size1, size2], device=dev)
     t1[3][0] = 1
+    t1[3][1] = 1
     # t2 has size [<=10, 2]
     t2 = torch.nonzero(t1)
     t5 = torch.ones(1, device=dev)
-    t5.expand(t2.size(0))
-    t5_cpu = t5.cpu()
-    self.assertEqual(t5_cpu.shape[0], 1)
+    t6 = t5.expand(t2.size(0))
+    t6_cpu = t6.cpu()
```

Comment:
I wonder why we need to do `.cpu()` first

Reply:
without the `.cpu` I expect the size to be a symint. Moving the tensor to cpu will trigger execution and materialize the size.